integration/integration_test.go

```diff
@@ -110,4 +108,22 @@ var _ = Describe("Client", func() {
 			Ω(entries).Should(ContainElement(BeEquivalentTo("Jumbo")))
 		})
 	})
+
+	Describe("PutIfAbsent", func() {
+		It("should write data to region only if absent", func() {
+			// do a put first
+			cluster.client.Put("FOO", "A", 777)
```

Comment:
This should be `PutIfAbsent` too I think.

Reply:
Changed it to be `PutIfAbsent` so it actually tests that it does put if absent.